lib/gcloud/search/field_values.rb

```diff
@@ -86,7 +86,10 @@ def [] index
       # Disabled because there are links in the docs that are long.
 
       ##
-      # Add a new value. The field name will be added to the value object.
+      # Add a new value. The field name will be added to the value object. If
+      # the field value is a DateTime or Numeric, or the type is set to
+      # +:datetime+ or +:number+, then the added value will replace any existing
+      # values of the same type.
```

Comment:
"any existing values of the same type" -> "an existing value of the same type" (since there can be only one)